src/utils/version.js

```diff
@@ -83,7 +83,7 @@ const version = {
    * @inner
    */
   isLedgerCustomTokenAllowed() {
-    const version = LOCAL_STORE.getWalletVersion();
```

Comment:
We were using the wallet version instead of the ledger version, which would always fail since the ledger version is ahead of the wallet